README.md

```diff
@@ -4,7 +4,15 @@ Site oficial da Angola Open-source Community
 
 ## Getting Started
 
-First, run the development server:
+First, install the dependencies:
```

Comment:
As instruções anteriores, são as instruções do framework. 

Existe uma razāo para mudar?

Reply:
777- Sim, aquelas instruções inicias é quando o projeto é criado, consequentemente vem com as dependências defaults.
Mas como o projecto está aqui no github, as dependências defaults e instaladas, elas não são submetidas no repositório.

Quem clonar o projecto na maquina local, e rodar por exemplo `npm run dev`,  retornaria  <b>erros</b>, porque as dependências estariam em falta.

Por isso, coloquei como o First: <b>install the dependencies</b>, para que, antes de rodar o servidor e evitar erros de dependências em falta, a pessoa instala as dependências executando o comando `npm install or yarn add`

Reply:
Que tal criamos uma PR diferente para isso?

Reply:
Tu precisas fazer fork do repositório. 
Adicionar as mudanças no teu fork do repositório e criar uma pull request.

Reply:
777- Já fiz isso...

Reply:
Talvez uma imagem do erro ia ajudar